test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,111 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('First Api Tests', () => {
+});
+
+
+it('Consume GET Service', () => {
```

Comment:
los it van dentro del describe

Reply:
Se realiza corrección y se incluyen los "it" dentro del describe.